kubevirt-disk-uploader.yaml

```diff
@@ -36,7 +36,7 @@ spec:
   serviceAccountName: kubevirt-disk-uploader
   containers:
     - name: kubevirt-disk-uploader
-      image: quay.io/boukhano/kubevirt-disk-uploader:latest
+      image: quay.io/boukhano/kubevirt-disk-builder:latest
```

Comment:
Can we make this pod PSA-friendly? https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted

This ensures it can be created in an arbitrary cluster-user namespace (which usually enforces the restricted policy)

Reply:
Thanks, I'll take a look.